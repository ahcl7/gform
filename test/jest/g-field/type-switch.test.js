import { generateComponent } from "./common";

describe('GField with type switch', function () {
  it('should render correctly', () => {
    const component = generateComponent({ key: 'switch', type: 'input', inputType: 'switch', flex: 'md-6' });
    expect(component.html()).toMatchSnapshot();
  });
});
