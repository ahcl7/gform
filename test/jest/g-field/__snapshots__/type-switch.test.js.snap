// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GField with type switch should render correctly 1`] = `
<div class="col-flex md-6 px-2">
  <div class="g-switch-wrapper">
    <label class="g-switch-container">
      <input type="checkbox">
      <span class="g-switch">
        <span class="g-switch-track">
        </span>
        <span class="g-switch-thumb">
        </span>
      </span>
    </label>
  </div>
</div>
`;
